src/validation/val/val.coffee

```diff
@@ -0,0 +1,40 @@
+validation = (validation) ->
+  auth = validation.auth
+  validation.validation =
+    add_team: (team) ->
+      return true
+    remove_team: (team) ->
```

Comment:
So this takes the result of a validation from validation/validate.coffee and returns that into the validation.validation.add_team? Just want to make sure this doesn't always return true whenever called unless any valid user can add or remove a team.